qiskit/pulse/commands/acquire.py

```diff
@@ -86,9 +86,9 @@ def __eq__(self, other: 'Acquire'):
         Returns:
             bool: are self and other equal.
         """
-        if type(self) is type(other) and \
-                self.kernel == other.kernel and \
-                self.discriminator == other.discriminator:
+        if (super().__eq__(other) and
```

Comment:
`==` doesn't work with `super()`?

Reply:
I don't think it will work, for the same reason we explicitly call `__init__` with super. [See for more info](https://stackoverflow.com/questions/25924548/why-do-we-have-to-use-the-dunder-methods-instead-of-operators-when-calling-v).